src/basis/FEBasisManagerDealii.h

```diff
@@ -61,35 +61,38 @@ namespace dftefe
 
       ////// FE specific  member functions /////
       void
-      reinit(const TriangulationBase &triangulation, const size_type feOrder);
+      reinit(const TriangulationBase &triangulation,
+             const size_type          feOrder) override;
       size_type
-      nLocalCells() const;
+      nLocalCells() const override;
       size_type
-      nGlobalCells() const;
+      nGlobalCells() const override;
       size_type
-      getFEOrder(size_type cellId) const;
+      getFEOrder(size_type cellId) const override;
       size_type
-      nCellDofs(size_type cellId) const;
+      nCellDofs(size_type cellId) const override;
       size_type
-      nLocalNodes() const;
+      nLocalNodes() const override;
       global_size_type
-      nGlobalNodes() const;
+      nGlobalNodes() const override;
       std::vector<size_type>
-      getLocalNodeIds(size_type cellId);
+      getLocalNodeIds(size_type cellId) const override;
       std::vector<size_type>
-      getGlobalNodeIds();
+      getGlobalNodeIds() const override;
```

Comment:
I think we shouldn't explicitly store the global node ids, only the range as this would require a large memory

Reply:
Ok based on our conversation, can we rename it to getLocallyOwnedGlobalNodeIds()?